helm/templates/job.yaml

```diff
@@ -45,7 +45,7 @@ spec:
       {{- toYaml . | nindent 8}}
     {{- end }}
       initContainers:
-        {{- if or (eq .Values.job.prefix "default") (eq .Values.job.prefix "attribute") }}
+        {{- if or (eq .Values.job.prefix "default") (eq .Values.job.prefix "attribute") (eq .Values.job.prefix "attribute-entity") }}
```

Comment:
should we keep the name as `data-config-service`? That will be consistent with the name of the service as well.

Reply:
Existing prefix names are `attribute` and `entity`, which seems to be denoting the bootstrap action. 
So `attribute-entity` seemed to be inline with the existing pattern.

Reply:
In that case, would `attribute-and-entity` make sense?

Reply:
done